src/Compilers/CSharp/Test/Semantic/Semantics/AwaitExpressionTests.cs

```diff
@@ -20,6 +20,97 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
     /// </summary>
     public class AwaitExpressionTests : CompilingTestBase
     {
+        [Fact]
+        public void TestAwaitInfoExtensionMethod()
+        {
+            var text =
+@"using System;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+static class App{
+    public static async Task Main(){
+        MyAwaitable Get()
+        {
+            return new MyAwaitable();
+        }
+
+        var x = Get();
```

Comment:
Is the local function required for the test? If not, perhaps `var x = new MyAwaitable();` instead.

Reply:
it is not required - I remove it